site/content/en/docs/user-manual/command-reference/describe-downloads.md

```diff
@@ -0,0 +1,53 @@
+---
+title: 'Describe downloadable datasets'
+linkTitle: 'describe-downloads'
+description: ''
+---
+
+This command reports reports various information about datasets that can be
+downloaded with the `download` command. The information is reported either as
+human-readable text (the default) or as a JSON object. The format can be selected
+with the `--report-format` option.
+
+When the JSON output format is selected, the output document has the following schema:
+
+```json
+{
+    "<dataset name>": {
+        "default_output_format": "<Datumaro format name>",
+        "description": "<human-readable description>",
+        "download_size": <total size of the downloaded files in bytes>,
+        "home_url": "<URL of a web page describing the dataset>",
+        "human_name": "<human-readable dataset name>",
+        "num_classes": <number of classes in the dataset>,
+        "subsets": {
+            "<subset name>": {
+                "num_items": <number of items in the subset>
+            },
+            ...
+        },
+        "version": "<version number>"
```

Comment:
Please indicate, who owns the version numbers - TFDS, we, dataset authors etc.

Reply:
I added a description.